.github/workflows/test.yml

```diff
@@ -0,0 +1,31 @@
+name: Run tests
+on: [pull_request]
+jobs:
+  test:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        python-version: [3.6, 3.7, 3.8, 3.9]
+
+    steps:
+      - uses: actions/checkout@v2
+
+      - name: Install Ubuntu packages
+        run: sudo apt-get install libxml2-dev libxslt1-dev python-dev
```

Comment:
Does the utils repo need this? I can't see any uses of `lxml`.

Otherwise looks sensible.